### ISSUE TITLE
more Verbose message

### ISSUE THREAD

**Issue by Imoteph** [PacificBiosciences/FALCON#70]
Hi @pb-jchin 

first of all, great pipeline!!! After applying successfully FALCON we got really nice results!

But I have to admit I had some very weird scenarios during the assembly process, where I really wished to know if FALCON is doing something or was waiting for a sub process to be finished. 

A issue was that FALCON was waiting for jobs to be finishing but "missed"  during the time that these jobs where done. So the main job was somehow stalling. The logs indicated that the jobs where finished. 
After restarting it went through. 

Another occasion where I thought FALCON was stalling again and I was nearly killing the jobs was when it was just creating the files/dir for 1-preads_oval. 

Therefore it would be nice, if FALCON can notify the current status on the cmd.

cheers

**Comment by pb-jchin** [comment]
check issue https://github.com/PacificBiosciences/FALCON/issues/43, I added some logging for job submission commit #b9257cb. If you are on a earlier revision, you might want to consider update to the most recent revision.

**Comment by Imoteph** [comment]
I'm using the up to date version, and have access to the sge_log and fc_run.log. 

from #43 this comment I really like to have seen in the log 

"3) when it is waiting on the done file of a job (just logs this
every 100 seconds)"

**Comment by pb-jchin** [comment]
It will need some communication between the remote processes and the `fc_run.py` hosting process. It could introduce more requirement for network communication and configuration to track the progress of each job. Such feature may make the installation and configuration more complicate. However, @pb-cdunn is thinking some alternative solution. If there is some elegant solution, we can put it in.

**Comment by pb-cdunn** [comment]
We have improved this on the *master* branch. Please create a new issue with any ideas for continued improvements.
